.gitignore

```diff
@@ -25,3 +25,6 @@ doc/_build
 
 # Generated man page
 doc/gitlab_backup.1
+
+.venv/
+.vscode/
```

Comment:
Revert these changes (and squash them out of your history). Thanks.

Reply:
Done